docker-config/Dockerfile

```diff
@@ -1,31 +1,20 @@
 FROM registry.digitalservice.id/proxyjds/library/alpine:edge
 
-RUN apk add chromium \
-      nss \
-      freetype \
-      harfbuzz \
-      ca-certificates \
-      ttf-freefont \
-      nodejs \
-      yarn
+RUN apk add nodejs yarn
 
 WORKDIR /app
 
-COPY . .
-
-ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true \
-    PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium-browser
+COPY package.json ./
 
-# Puppeteer v6.0.0 works with Chromium 89.
 RUN yarn install
 
-# Add user so we don't need --no-sandbox.
+COPY . .
+
 RUN addgroup -S pptruser && adduser -S -g pptruser pptruser \
-    && mkdir -p /home/pptruser/Downloads /app /app/tmp \
+    && mkdir -p /app /app/tmp \
     && chown -R pptruser:pptruser /home/pptruser \
     && chown -R pptruser:pptruser /app
 
-# Run everything after as non-privileged user.
 USER pptruser
 
 EXPOSE 3003
```

Comment:
[OPTIONAL] 
can this port number be taken from the env file? 😄

Reply:
yes, i was update here 56aed0c